sections/chla_post_recovery.md

```diff
@@ -4,13 +4,21 @@
 At first users should report that their mission is over to support(at)oceanops.org
 
 ## Biofouling assessment
-...
+Pictures should be taken for biofuiling assessment. 
+Do we have good photos of biofouling? 
+
 
 ## Storage and cleaning
-...
+After recovery and once all caibrations are done, clean the sensor and remove any biofouling. The following protocol is recommended by SeaBird for the Wetlabs EcoPUCK:
+
+1. Remove the protective copper plate, if present.
+2. To clean the optics head use soap and water with a non abrasive pad.
+3. If necessary, a wooden scraper can be used to push the material from the optics area.
+4. Rinse very well with clean water and dry carefully.
+5. Do not use any strong solvent (ie. acetone or MEK) which could ruin the epoxy head of the sensor
 
 ## Lab calibration
-...
+When the glider is recovered, tape and check for dark count stability or use the seabird calibration stick?
 
 ## Field calibration
-...
+If you recover the glider from a small boat or a research vessel follow protocols described in {numref}`deployment_small_boat` and {numref}`deployment_ship_CTD`, respectively.
```

Comment:
Are these meant to be links? They don't lead anywhere

Reply:
ok, so in mission execution section we need to add headings the little = incantation so that numref will find and make the links like they have here on line 66-67 https://github.com/OceanGlidersCommunity/Oxygen_SOP/blob/2dff6bf91ec28a553c898ac7d32bb51e4728ea3d/sections/oxygen_pre_deployment.md?plain=1#L66

May as well do it in this PR so that the links are created properly, even if the sections for small boat deployment will be empty for now

Reply:
Ok ok